server/chat-commands/moderation.ts

```diff
@@ -572,18 +598,18 @@ exports.commands = {
 	ipmute: 'lock',
 	wl: 'lock',
 	weeklock: 'lock',
-	monthlock: 'lock',
```

Comment:
?

Reply:
must have missed this while implementing monthlock in my fork, mb